hdat/resultspec.py

```diff
@@ -48,9 +67,9 @@ def resolve_resultspecs(archive, resultspec):
         else:
             return [result]
     elif pick_recents_all_suites:
-        return archive.select_recents_all()
```

Comment:
See comment about adding `suites` to the `Archive` constructor. Some stuff would have to be switched around in `main.py` but I think it's possible.